application/usecases/instagram_posts.go

```diff
@@ -0,0 +1,164 @@
+package usecases
+
+import (
+	"TML_TBot/config"
+	"TML_TBot/domain/models"
+	"bytes"
+	"fmt"
+	"image"
+	"image/png"
+	"net/http"
+	"os"
+	"reflect"
+
+	"github.com/Davincible/goinsta/v3"
+)
+
+type InstagramPostsController struct {
+	feedItems []FeedItem
```

Comment:
Seems this attr is not being used
```suggestion
```

Reply:
I meant the struct attribute, feedItems (FeedItem it's being used once indeed)

Reply:
True, removed